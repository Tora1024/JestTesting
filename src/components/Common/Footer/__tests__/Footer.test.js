import React from 'react';
import { shallow } from 'enzyme';
import ComponentUnderTest from '../Footer';

describe('components/Footer/Footer.js', () => {
  let wrapper;

  describe('in general', () => {
    beforeEach(() => {
      wrapper = shallow(<ComponentUnderTest />);
    });
    it('should render correct snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
