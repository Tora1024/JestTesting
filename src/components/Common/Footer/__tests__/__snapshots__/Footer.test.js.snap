// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Footer/Footer.js in general should render correct snapshot 1`] = `
<footer
  className="footer"
>
  <nav
    className="nav"
  >
    <Link
      className="nav-item"
      replace={true}
      to="/"
    >
      Home
    </Link>
    <Link
      className="nav-item"
      replace={true}
      to="/about"
    >
      About
    </Link>
  </nav>
  <section
    className="social"
  >
    <div
      className="facebook"
    />
    <div
      className="linkedin"
    />
    <div
      className="skype"
    />
    <div
      className="github"
    />
  </section>
  <section
    className="copyright"
  >
    Created by @HÃ©ctor
  </section>
</footer>
`;
